db/celebrity.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const uniqueValidator = require('mongoose-unique-validator');
+
+var Schema = mongoose.Schema;
+
+//Star schema
+var CelebSchema = new Schema({
+   name: {type: String, required: true, unique: true},
+   occupation: String,
+   albums: Number
+});
+
+CelebSchema.plugin(uniqueValidator);
+
+var Celeb = mongoose.model('Celeb', CelebSchema, 'Celebrities');  
```

Comment:
Let's use the full term Celebrity, also through out application use singular term (i.e., no s)
so this line must be
`var Celebrity = mongoose.model('Celebrity', CelebritySchema, 'Celebrity'); `